code/datums/limb.dm

```diff
@@ -1444,6 +1444,14 @@ var/list/ghostcritter_blocked = ghostcritter_blocked_objects()
 	max_wclass = 3
 	stam_damage_mult = 1
 
+/datum/limb/small_critter/pincers
```

Comment:
should have 
`dmg_type = DAMAGE_CUT` or `DAMAGE_STAB` (your pick I don't really care)